templates/assets/js/toggleSidebar.js

```diff
@@ -12,8 +12,16 @@ function toggle() {
 
 $(window).resize(() => {
   const w = $(this).width();
-  if (w < 576) {
+  if (w < 992) {
     document.querySelector(".sidebar").classList.remove("visible");
     document.querySelector(".sidebar").classList.add("hidden");
   }
 });
+
+$(document).ready(() => {
+  const w = $(this).width();
+  if (w > 992) {
+    document.querySelector(".sidebar").classList.remove("hidden");
+    document.querySelector(".sidebar").classList.add("visible");
```

Comment:
Isn't there a better way to switch visibility? A single function?

Reply:
Actually there is no need of visible class, but since it was present in the logic, I didn't removed it.
But yes, visible class is empty and not required. 
I would remove it if you allow.

Reply:
Yes please remove it